packages/flutter/lib/src/cupertino/bottom_tab_bar.dart

```diff
@@ -240,10 +239,14 @@ class CupertinoTabBar extends StatelessWidget implements PreferredSizeWidget {
                   tabCount: items.length,
                 ),
                 child: MouseRegion(
-                  cursor:  kIsWeb ? SystemMouseCursors.click : MouseCursor.defer,
+                  cursor: kIsWeb ? SystemMouseCursors.click : MouseCursor.defer,
                   child: GestureDetector(
                     behavior: HitTestBehavior.opaque,
-                    onTap: onTap == null ? null : () { onTap!(index); },
+                    onTap: onTap == null
```

Comment:
This time it did move the ternary onto two lines 🤔

Reply:
This is because of the function with body. dartfmt always put body on a new line. I think a one liner will be used with an arrow function `() => onTap!(index)`